W-21917743 apim relaunch may 11 jh (#355)
mulesoft/docs-exchange: modules/ROOT/pages/adding-kong-gateway-scanner.adoc

= Adding a Scanner for Kong Gateway
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> Add a scanner to discover, import, and sync APIs from Kong Gateway into Exchange. Then you can govern the APIs and consume them in other applications.

Reviewer: @JennyHajee 
```suggestion
Add a scanner to discover, import, and sync services from Kong Gateway into Exchange. Once imported, these services are available for governance, policy enforcement, and exposure through your developer portal.
```

> Add a scanner to discover, import, and sync services from Kong Gateway into Anypoint Exchange. Once imported, you can apply governance, enforce policies, and publish them to your developer portal.

Reviewer: ```suggestion
Add a scanner to discover, import, and sync APIs from Kong Gateway into Anypoint Exchange. Once imported, you can apply governance rulesets and publish them to your developer portal.
```

== Before You Begin
Before adding the scanner, verify that you have the permission, context, and these Kong credentials:

* Exchange Administrator permission 

> * Kong Gateway region

Reviewer: ```suggestion
*Kong Control Plane Viewer permission 

* Kong Gateway region
```

* Personal access token (PAT)

== Add a Scanner for Kong Gateway
. Verify that you are in the business group where you want to add the scanner.
. From the sidebar in Exchange, click *Scanners*.
. Enter a name for the scanner.
. From *Scanner Run Configuration*, complete these fields or options:
+
[%header,cols="1,4"]
|===
|*Field/Option* |*Value*
|*Run Schedule* |Select a frequency and local time.
|*Sync Review* |Select an option: Auto-resolve or Ask to review.
|===

. From *Connection Configuration*, complete these fields:
+
[%header,cols="1,4"]
|===
|*Field* |*Value*
|*Provider* |Select *Kong*.
|*Platform* |Select *Kong Gateway*.

> |*Service Type* |Select *REST APIs*.

Reviewer: @JennyHajee 
```suggestion
|*Service Type* |*REST APIs* will be preselected.
```

> |*Authentication Method* |Select *Service Account*.

Reviewer: @JennyHajee 
```suggestion
|*Authentication Method* |*Service Account* will be preselected.
```
